backend/src/main/java/com/onair/hearit/admin/infrastructure/s3/S3Config.java

```diff
@@ -12,10 +12,18 @@ public class S3Config {
     @Value("${aws.region}")
     private String region;
 
+    @Value("${aws.s3.bucket}")
+    private String bucket;
+
     @Bean
     public S3Client s3Client() {
         return S3Client.builder()
                 .region(Region.of(region))
```

Comment:
```suggestion
                .region(Region.AP_NORTHEAST_2)
```

제가 이거 수정하는거 깜빡했는데 S3에서 Region을 제공하더라고요. 그냥 제공해주는 거 써도 될 듯요 딱히 민감정보도 아니고

Reply:
변경했습니다!